### ISSUE TITLE
QEMU Full System Example Fuzzers Are Broken

### ISSUE THREAD

**Issue by riesentoaster** [AFLplusplus/LibAFL#2785]
`qemu_linux_system` and `qemu_linux_process` throw an error on `cargo make build`.

- For `qemu_linux_system`, the import path to other repo parts are broken.
- For both, I think some of the features are messed up.
- Other things may be broken as well.

Whoever fixes this: Consider replacing all `#[allow(…)]` calls with `#[expect(…)]` like in #2784.

**Comment by riesentoaster** [comment]
Also: Consider adding them to the CI, so this doesn't happen again.

**Comment by rmalmain** [comment]
yeah, this is a known issue. we put these examples for reference atm, but it requires a repository that is not public yet.
we plan to release them in the near future.

**Comment by riesentoaster** [comment]
@tokatoka @rmalmain Is this actually fixed? I'm still getting errors when building `qemu_linux_process` and `qemu_linux_kernel`.

**Comment by tokatoka** [comment]
Hi. As romain mentioned earlier, the current state of qemu_linux_process is intentional. It doesn't work without a private repo which is hidden right now but is planned to be publicized once we finish writing the paper.

If you think it should be kept open feel free to reopen this issue 👍

**Comment by riesentoaster** [comment]
Fair enough. I was just surprised it got closed out of the blue. There's a note in the README, which I previously missed, that's probably enough. 

Looking forward to reading what you've been up to :)
